libs/coin-modules/coin-tezos/src/bridge/prepareTransaction.integ.test.ts

```diff
@@ -0,0 +1,49 @@
+import BigNumber from "bignumber.js";
+import { createFixtureAccount, createFixtureTransaction } from "../types/bridge.fixture";
+import prepareTransaction from "./prepareTransaction";
+import coinConfig, { TezosCoinConfig } from "../config";
+
+describe.only("prepareTransaction", () => {
+  beforeAll(() => {
```

Comment:
I think we should either mock returned value or set back previous value in afterAll clause

Not sure if it runs in same thread as other but i find it scary to edit coinconfig globally

Reply:
For the time being, integration test in CoinModule are running only on-demand. Meaning that you have to launch them on your laptop.
We definitely need a task for those tests and think about some global setup for CoinConfig for each coins.